### ISSUE TITLE
[bug][2018.4.0][Backend]Unable to create Locales in the backend.

### ISSUE THREAD

**Issue by guelzow** [aimeos/aimeos-typo3#68]
Hi everyone.

I'm not sure, if this is the right bug tracker to report this one.

When
- creating a new site
- and then trying to create a new Locales combination

the backend forgets for which environment you want to create these, resulting in a disabled form.

**Our setup:**
TYPO3 8.7.10
TYPO3 Aimeos extension 18.4.0

**What is happening in the Code:**
It took a little bit to track this one.
https://github.com/aimeos/aimeos-core/blob/master/lib/mshoplib/src/MShop/Locale/Manager/Standard.php#L547
Tries to load the Locales from the DB and finds nothing. It then throws an exception.

The exception then gets caught in
https://github.com/aimeos/aimeos-typo3/blob/master/Classes/Base/Locale.php#L88
creating a new, but empty Item.

Then the template file locale/item-standard.php relies on the loaded Locales, but finds nothing.
The form is then disabled, hidden fields do not get set.
Trying to save the form results in a SQL error.

**Our workaround so far:**
The Locale\Manager returns a false in
https://github.com/aimeos/aimeos-core/blob/master/lib/mshoplib/src/MShop/Locale/Manager/Standard.php#L591
when no Locales where found at all.
With a
```
$row = array();
$row['locale.siteid'] = $siteId;
return $this->createItemBase( $row, $siteItem, $sitePath, $siteSubTree );
```
we are at least able to give feedback about the current site to the backend, so we can create the new Locales.

Tobi

**Comment by aimeos** [comment]
Can you please post some screenshots with the steps to reproduce? If we

- go to the locale site panel
- create a new site
- save the new site
- switch to the new site in the top left site selector
- go to the local panel
- create a new locale combination
- save the new locale combination

everything works as expected

**Comment by guelzow** [comment]
Created a new site
![01 - created a new site](https://user-images.githubusercontent.com/11192910/39954324-c494d7a4-55bd-11e8-9e59-d14c729e464c.png)

List of all sites
![02 - list all sites](https://user-images.githubusercontent.com/11192910/39954326-cf4f80f4-55bd-11e8-9f1a-5d7a39f18b9c.png)

List of all Locales of the new site
![03 - list all e-brochure locales](https://user-images.githubusercontent.com/11192910/39954328-d83413c4-55bd-11e8-99b8-393eb3322909.png)

Disabled Locales form (save button can still be clicked, though)
![04 - new locales disabled form](https://user-images.githubusercontent.com/11192910/39954332-e542d4ce-55bd-11e8-98f3-c438ea85114e.png)

SQL error on save, due to missing hidden fields.
![05 - sql error when saving](https://user-images.githubusercontent.com/11192910/39954336-fb30996a-55bd-11e8-92e1-92fd03c00dda.png)

Analysis of the empty Locales item, which is created in https://github.com/aimeos/aimeos-typo3/blob/master/Classes/Base/Locale.php#L89
![06 - analysis of the loaded site](https://user-images.githubusercontent.com/11192910/39954349-3661afec-55be-11e8-99ae-625db03a7ed9.png)

**Comment by aimeos** [comment]
Thank you, the problem has been fixed and will be part of the next release: df2cc871216546646641384cfbebd05465ad2c1a

Please remember that you need to execute the update script in the Extension Manager nevertheless to create the necessary types records.
